kubernetes-ts-multicloud/README.md

```diff
@@ -0,0 +1,82 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# Kubernetes Multi-cloud Example
```

Comment:
When I saw the "k8s multi-cloud" title, I thought I would see one cluster spanning multiple clouds. Is it just me or may it be confusing?

Reply:
Good feedback - will clarify